portfolio/src/components/conteudo/index.js

```diff
@@ -0,0 +1,2 @@
+import Conteudo from "./conteudo"
+export default Conteudo;
```

Comment:
Gostei da ideia de fazer um arquivo index para facilitar os imports. 🚀 

Uma dica que acho que pode facilitar sua vida:
Ao invés de exportar como `default` vc pode exportar apenas a componente pelo `export`. Exemplo:

> // arquivo exemplo react-component.tsx
> export function ReactComponent() {
>     return <></>;
> }

Note que você pode adicionar apenas o `export` antes da função/classe/constante que deseja exportar. Na hora de importar, é apenas necessário acessar o nome da função que está sendo exportada:

> //arquivo app.tsx
> import { ReactComponent } from './path-to-component/react-component';
> 
> function App() {
>   return <ReactComponent/>
> }

Perceba que agora não podemos chamar o componente com outro nome além que foi dado (tem q importar o ReactComponent, nao da pra chamar ele de outro nome), então é apenas uma mudancinha que pode facilitar um pouco sua vida. No arquivo index.ts vc pode fazer apenas:

> export * from './react-component'; 

e o import passa a ser:

> import { ReactComponent } from './path-to-component';

Obs: Se quiser renomear a funcao dentro do import vc pode fazer:
`import { ReactComponent as AnyOtherName } from './path-to-component';`